src/content/reference/react/useDebugValue.md

```diff
@@ -103,20 +103,20 @@ function subscribe(callback) {
 
 <Note>
 
-Don't add debug values to every custom Hook. It's most valuable for custom Hooks that are part of shared libraries and that have a complex internal data structure that's difficult to inspect.
+Her özel Hook'a hata ayıklama değeri eklemeyin. En değerli olanı, paylaşılan kütüphanelerde bulunan ve incelenmesi zor olan karmaşık bir iç veri yapısına sahip olan özel Hook'lar için geçerlidir.
```

Comment:
`valuable` icin `degerli`den ziyade `faydali` daha dogru sanki. Ikinci cumle icin bu daha iyi bir ceviri olabilir: 

`Paylaşılan kütüphanelerde bulunan ve incelenmesi zor olan karmaşık bir iç veri yapısına sahip olan özel Hook'lar için kullanıldığında daha faydalıdır. `